scripts/ci/prek/update_airflow_pyproject_toml.py

```diff
@@ -182,13 +182,7 @@ def get_python_exclusion(provider_dependencies: dict[str, Any]) -> str:
         else:
             all_optional_dependencies.append(f'"{optional}" = [\n    "apache-airflow-core[{optional}]"\n]\n')
     optional_airflow_task_sdk_dependencies = get_optional_dependencies(AIRFLOW_TASK_SDK_PYPROJECT_TOML_FILE)
-    for optional in sorted(optional_airflow_task_sdk_dependencies):
-        if optional == "all":
-            all_optional_dependencies.append('"all-task-sdk" = [\n    "apache-airflow-task-sdk[all]"\n]\n')
-        else:
-            all_optional_dependencies.append(
-                f'"{optional}" = [\n    "apache-airflow-task-sdk[{optional}]"\n]\n'
-            )
+    all_optional_dependencies.append('"all-task-sdk" = [\n    "apache-airflow-task-sdk[all]"\n]\n')
```

Comment:
Hmm.. One small thing here = the idea here was that we should automatically update the pyproject.toml with "all" - having all the other manually defined extras. while the original code was somewhat wrong, I think it needs better fix. Generally it should read all extras defined, and it should automatically generate

```
"all" = [ 
    "apache-airflow-task-sdk[ALL OTHER EXTRAS DEFINED HERE]" 
```

That was the intention (even if original code was not doing it because there was only one extra defined - it did its work.